File: frontend/src/app/person.ts
import {Address} from './address';

export class Person {
  id: string;
  name: string;
  email: string;
  phoneNumber: string;
  address: Address;

  constructor(
    id: string,
    name: string,
    email: string,
    phoneNumber: string,
    address: Address = null,
  ) {
    this.id = id;
    this.name = name;
    this.email = email;
    this.phoneNumber = phoneNumber;
    this.address = address;
  }
}

File: backend/ExampleAPI/ExampleAPI2/Models/Person.cs
namespace ExampleAPI2.Models
{
    public class Person
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public string PhoneNumber { get; set; }
        public Address Address { get; set; }
    }
}
